tests/sentry/incidents/endpoints/test_organization_alert_rule_details.py

```diff
@@ -60,6 +64,39 @@ class AlertRuleDetailsBase(AlertRuleBase):
 
     endpoint = "sentry-api-0-organization-alert-rule-details"
 
+    @cached_property
+    def dynamic_alert_rule_dict(self):
```

Comment:
Is this the same as in `test_organization_alert_rule_index`? If so could you put it in a shared place so it's not duplicated?

Reply:
We actually don't need to duplicate it since `AlertRuleDetailsBase` inherits `AlertRuleBase` 😅